Dockerfile.debos-arch

```diff
@@ -0,0 +1,11 @@
+ARG BASE_REGISTRY=ghcr.io/go-debos/test-containers
+ARG BASE_IMAGE=fakemachine-arch
+ARG BASE_TAG=latest
+FROM $BASE_REGISTRY/$BASE_IMAGE:$BASE_TAG
+
+# Bits needed to run debos test suite
+RUN pacman -Syu --noconfirm dpkg \
+                            unzip
```

Comment:
Side note: We should explore switching to libarchive and it's tools - bsdtar, etc
 - single tool to interact with - supporting dpkg, zip, rar, nearly every tar permutation...
 - more widely available due to its BSD license
 - always present and working - as Arch is concerned - our package manager (pacman) depends on libarchive

Reply:
right - can you open an issue on debos so that doesn't get lost ? :-)

Reply:
Good call - forwarded to https://github.com/go-debos/debos/issues/316